README.md

```diff
@@ -1,19 +1,19 @@
 [![AsyncAPI Design System](/assets/github-repobanner-designsystem.png)](https://www.asyncapi.com)
 
-## Environment Setup
+# :deciduous_tree:Environment Setup
 
-### How to setup storybook locally
+### :joystick:How to setup storybook locally
 
-- Before starting the Storybook environment locally, install all the required dependencies. Run the command below to install the dependencies: 
+- Before starting the Storybook environment locally, install all the required dependencies. Run the command below to install the dependencies:
 
 ```cmd
-npm install --save-dev
+  npm install --save-dev
```

Comment:
Try to make good indents by adding two spaces before "`" signs and also change

```
npm install --save-dev
```

to the

```
npm install
```